ExampleApp/Views/BaseViewController.swift

```diff
@@ -0,0 +1,194 @@
+//
+//  BaseViewController.swift
+//  OmiseSDK
+//
+//  Created by Andrei Solovev on 4/10/23.
+//  Copyright © 2023 Omise. All rights reserved.
+//
+
+import UIKit
+import OmiseSDK
+
+enum CodePathMode: Int {
+    case storyboard = 0
+    case code = 1
+}
+
+class BaseViewController: UIViewController {
+
+    @IBOutlet private var modeChooser: UISegmentedControl!
+
+    var currentCodePathMode: CodePathMode = .code
+    var paymentAmount: Int64 = 0
+    var paymentCurrencyCode: String = ""
+    var usesCapabilityDataForPaymentMethods = false
+    var allowedPaymentMethods: [SourceTypeValue] = []
+
+    required init?(coder: NSCoder) {
+        self.usesCapabilityDataForPaymentMethods = true
+        super.init(coder: coder)
+    }
+
+    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        self.usesCapabilityDataForPaymentMethods = true
+        super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
+    }
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Workaround of iOS 12 bug on the tint color
+        self.view.tintColor = nil
+        self.navigationController?.navigationBar.tintColor = nil
+
+        updateUIColors()
+
+        #if INTERNAL_TEST
+        modeChooser.selectedSegmentIndex = CodePathMode.code.rawValue
+        modeChooser.isEnabled = false
+        #endif
+
+        let localeCountryCode: String? = {
+            if #available(iOS 16, *) {
+                return Locale.current.region?.identifier
+            } else {
+                return Locale.current.regionCode
+            }
+        }()
+
+        switch localeCountryCode {
+        case "JP":
+            self.paymentAmount = Tool.japanPaymentAmount
+            self.paymentCurrencyCode = Tool.japanPaymentCurrency
+            self.allowedPaymentMethods = Tool.japanAllowedPaymentMethods
+        case "SG":
```

Comment:
What about EN locale? Is it supposed to be same as SG and hence just ommitted?

Reply:
There's a `default` block for everything else includes EN just below:

```
default:
    self.paymentAmount = Tool.thailandPaymentAmount
    self.paymentCurrencyCode = Tool.thailandPaymentCurrency
    self.allowedPaymentMethods = Tool.thailandAllowedPaymentMethods
```